_episodes/06-script.md

```diff
@@ -550,7 +558,7 @@ Of course, this introduces another tradeoff between flexibility and complexity.
 
 > ## Script Reading Comprehension
 >
-> For this question, consider the `shell-lesson-data/molecules` directory once again.
+> For this question, consider the `shell-lesson-data/exercise-data/proteins` directory once again.
```

Comment:
This one is OK, but there are still a number of references to `molecules` in this lesson. Probably just need to find/replace those.